### ISSUE TITLE
Update input helper text to meet AAA contrast guidelines

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2751]
### Issue description

The input helper text does not meet AAA color contrast guidelines.

### Acceptance criteria

- [ ] Choose a darker color for the helper text
- [ ] Create a ticket to update helper text color in the figma design system
- [ ] Create a dev ticket to change the helper text color in the code

### Additional context

This does not conform with our accessibility target, WCAG 2.1.

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2849

**Comment by SamiyahKey** [comment]
### Prioritization Comments

**WCAG Standard in question:** [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)
**Prioritization:** Normal
**Reason:** Although we should strive to meet AAA guidelines for contrast, this is not a requirement. The contrast currently passes AA guidelines and is not a critical failure for users. 

![Image](https://github.com/user-attachments/assets/15d73f65-c121-4038-be6b-0f5ef48c990b)
